wsagent/wsagent-local/src/main/java/org/eclipse/che/MachinePublicKeyProvider.java

```diff
@@ -29,20 +29,24 @@
  */
 public class MachinePublicKeyProvider implements Provider<PublicKey> {
 
+  private final String publicKey;
+  private final String algorithm;
+
+  @Inject
+  public MachinePublicKeyProvider(
+      @Named("env.CHE_MACHINE_AUTH_SIGNATURE__ALGORITHM") String alg,
+      @Named("env.CHE_MACHINE_AUTH_SIGNATURE__PUBLIC__KEY") String key) {
+    checkArgument(key != null && !key.isEmpty(), "Signature key for token validation is not found");
```

Comment:
AFAIK env vars injected using this way don't support not set env vars. Have you tried it?

Reply:
I'll check that, thx